src/modules/Dashboard/features/RespondentData/RespondentDataHeader/RespondentDataHeader.tsx

```diff
@@ -0,0 +1,170 @@
+  };
+
+  const handleOpenExport = () => {
+    setIsExportOpen(true);
+    Mixpanel.track('Export Data click');
+  };
+
+  const handleCloseExport = () => {
+    setIsExportOpen(false);
+  };
+
+  const canDoTakeNow =
+    activity &&
+    featureFlags.enableMultiInformantTakeNow &&
+    (isManagerOrOwner(roles?.[0]) || roles?.includes(Roles.SuperAdmin));
+
+  const canViewData = hasPermissionToViewData(roles);
+
+  const headerElements = {
+    name: activity ? activity.name : applet.displayName,
+    image: activity ? activity.image : applet.image,
+  };
+
+  return (
+    <>
+      <StyledFlexTopStart
+        sx={{
+          margin: theme.spacing(1.2, 3.2, 3.2),
+        }}
+      >
+        <NavigationEyebrow title={t('back')} subtitle={'Test'} onClick={navigateUp} />
+      </StyledFlexTopStart>
+
+      <StyledFlexSpaceBetween
+        sx={{
+          gap: theme.spacing(1.6),
+          margin: theme.spacing(0, 3.2, 0.8),
+        }}
+      >
+        <StyledFlexTopCenter
+          sx={{
+            gap: theme.spacing(1.6),
+          }}
+        >
+          {!!headerElements.image && <StyledLogo src={headerElements.image} />}
+          <StyledHeadlineLarge color={palette.on_surface}>
+            {headerElements.name}
+          </StyledHeadlineLarge>
+          <Chip
+            icon={<Svg id="respondent-circle" width={18} height={18} />}
+            color={'secondary'}
+            shape={ChipShape.Rectangular}
+            sx={{ py: 0.5, height: 'auto', px: 1, alignItems: 'end' }}
+            title={subject?.secretUserId || ''}
+          />
+        </StyledFlexTopCenter>
+        {canViewData && (
+          <StyledFlexTopCenter sx={{ gap: 1 }}>
+            <Button
+              data-testid={`${dataTestid}-export-button`}
+              onClick={handleOpenExport}
+              startIcon={<Svg id="export" width={18} height={18} />}
+              sx={{ color: variables.palette.on_surface_variant }}
+            >
+              {t('export')}
+            </Button>
+
+            <ExportDataSetting
+              isExportSettingsOpen={isExportOpen}
+              onExportSettingsClose={handleCloseExport}
+            />
+            {featureFlags.enableActivityAssign && (
+              <ActionButton
+                onClick={handleAssignActivity}
+                data-testid={`${dataTestid}-assign-activity`}
+                sx={{ backgroundColor: variables.palette.secondary_container }}
+              >
+                {t('assign')}
+              </ActionButton>
+            )}
+            {canDoTakeNow && (
+              <ActionButton
+                variant="contained"
+                onClick={handleTakeNow}
+                data-testid={`${dataTestid}-take-now`}
+                sx={{
+                  backgroundColor: variables.palette.primary,
+                  color: variables.palette.white,
+                }}
+              >
+                {t('takeNow')}
+              </ActionButton>
```

Comment:
We shouldn't have to apply custom colours to buttons anymore as our variants take care of this. Instead you could use `sx` for the `minWidth`:

```tsx
<Button variant="tonal" sx={{ minWidth: '10rem' }} …>

<Button variant="contained" sx={{ minWidth: '11.3rem' }} …>
```

An afterthought: We've been appying `minWidth` to buttons a fair bit, and looking at Figma, I think all buttons should actually have a default min width of `10rem`. Also the horizontal padding we have set in the theme might not be set right, it should be 48px. Those changes might bring the second button to the right size naturally. Using `minWidth` on a per-button basis feels a bit hacky considering the text is going to be a different width in French… Anyway, I get if this is out of scope, just thought to mention it.

Reply:
Great suggestion, I confirmed we no longer need to set `minWidth` as that was the result of using the base `Button`.